src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -245,32 +246,10 @@ class GTGRunner(
     })
   }
 
-  /**
-   * Collect the edges of the form ((String, Double), (String, Double))
-   * From the edges collect all used vertices.
-   * Repeated vertices are eliminated due to the set conversion.
-   * @param MCCEdgeList Collection of MCCEdges
-   * @param MCCNodeMap Dictionary of all the MCCNodes
-   */
-  def processEdges(MCCEdgeList: Iterable[MCCEdge], MCCNodeMap: mutable.HashMap[String, MCCNode]): Unit = {
-    logger.info("NUM VERTICES : " + MCCNodeMap.size + "\n")
-    logger.info("NUM EDGES : " + MCCEdgeList.size + "\n")
-
-    val pw = new PrintWriter("MCCNodesLines.json")
-    MCCNodeMap.foreach { case (key, value) =>
-      pw.write(value.toString())
-      pw.write("\n")
-    }
-    pw.close()
-
-    val fw = new PrintWriter("MCCEdges.txt")
-    fw.write(MCCEdgeList.toString())
-    fw.close()
-  }
-
   def run(): Unit = {
 
     logger.info("Starting MCC")
+    val outputDir = MCSUtils.testHDFSWrite(this.outputDir)
```

Comment:
In terms of naming convention, please rename this function. In the codebase functions beginning with 'test' are found in the test dir.